htdocs/core/class/extrafields.class.php

```diff
@@ -1947,10 +1947,6 @@ public function setOptionalsFromPost($extralabels, &$object, $onlykey = '')
 				if (!empty($onlykey) && $onlykey != '@GETPOSTISSET' && $key != $onlykey) {
 					continue;
 				}
-				if (!empty($onlykey) && $onlykey == '@GETPOSTISSET' && !GETPOSTISSET('options_'.$key) && $this->attributes[$object->table_element]['type'][$key] != 'boolean') {
```

Comment:
This will break other feature. Can you try by replacing
`$this->attributes[$object->table_element]['type'][$key] != 'boolean'`
with
`(! in_array($this->attributes[$object->table_element]['type'][$key], array ('boolean', 'chkbxlst'))`

Reply:
Hello @eldy tks for your help !
Ok I try and run on my locale this fix.
If it works I will update my PR.